rpc/daemon/daemon.proto

```diff
@@ -52,5 +53,6 @@ message TableName {
 }
 
 message ManagerInfo {
-  int32 grpc_port = 1;
+  manager.SessionInfo session = 1;
+  int32 grpc_port = 2;
```

Comment:
Probably
```protobuf
  manager.SessionInfo session = 2;
  int32 grpc_port = 1;
  ```
  to avoid re-numbering grpc_port?

Reply:
I fixed this with a squash while rebasing.